src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card card--size" data-qa="card">
+      <div class="card__photo">
+        <img
+          src="images/imac.jpeg"
+          alt="photo"
+          class="card__img"
+        >
+      </div>
+      <h3 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h3>
+      <p class="card__code">Код товара: 195434</p>
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__feedback">Отзывов: 5</p>
+      </div>
+      <div class="card__price">
+        <p class="price__word">Цена:</p>
+        <p class="price__amount">69 999 грн</p>
```

Comment:
У тебя нет блока price, поэтому ты не можешь делать его элемент price__word.
```suggestion
      <div class="card__price">
        <p class="card__price-name">Цена:</p>
        <p class="card__amount">69 999 грн</p>
```